inference-engine/tests/ngraph_functions/include/ngraph_functions/builders.hpp

```diff
@@ -220,5 +220,15 @@ std::shared_ptr<ngraph::Node> makeSpaceToDepth(const ngraph::Output<Node> &in,
                                                ngraph::opset3::SpaceToDepth::SpaceToDepthMode mode,
                                                size_t blockSize);
 
+std::shared_ptr<Node> makePooling(const ngraph::Output<Node> &in,
+                                  const std::vector<size_t> &strides,
+                                  const std::vector<size_t> &padsBegin,
+                                  const std::vector<size_t> &padsEnd,
+                                  const std::vector<size_t> &kernel,
+                                  const op::RoundingType &roundingType,
+                                  const op::PadType &padType,
+                                  bool excludePad,
```

Comment:
Nitpick: I don't really like `bool` parameters for a functions to modify behavior, it's unclear what it means on call side:

```
makePooling(..., true, ...);
makePooling(..., false, ...);
// guess what's the difference
```

But since it's internal API for testing, I think it's ok. At least it's reasonable to fix that in a separate request, not now..